docs/rules/camelcase.md

```diff
@@ -32,9 +32,27 @@ obj.do_something = function() {
     // ...
 };
 
+function foo({ no_camelcased }) {
+    // ...
+};
+
+function foo({ no_camelcased: my_default }) {
```

Comment:
I wonder if this example would be clearer as something like:
`function foo({ isCamelcased: no_camelcased }) {`